lib/build_task.rb

```diff
@@ -22,6 +22,12 @@ def build(system_description, output_path, options = {})
     LocalSystem.validate_existence_of_package("kiwi")
     LocalSystem.validate_existence_of_package("kiwi-desc-vmxboot")
 
+    if system_description.os.architecture != "x86_64"
```

Comment:
Shouldn't we extend the os class to also have information regarding the can_build architecture and not just the distribution?